### PR TITLE
Bug 979827 - [Build] check node version of environment.

### PR DESCRIPTION
**Pull request by cctuan**
https://bugzilla.mozilla.org/show_bug.cgi?id=979827

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b10a000c7de2a8da424d8e5cac8f894c7f2cc658

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=261826b448ff727a3b648f034b95fcd9d281914f

File: Makefile [hunk 1]
```diff
@@ -124,6 +124,9 @@ RAPTOR?=0
 # Share performance and usage data
 SHARE_PERF_USAGE?=1
 
+# what major version of node we expect to run?
+NODE_VERSION?=v0.10
```
Comment (rickychien) [inline Makefile]: I don't think it's necessary to specify it by user input. Just ```NODE_VERSION=v0.10```

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6278fe93f250e338696af38e865d9b644e5d1579

**Pull request merged**